AmplifyPlugins/API/AWSAPICategoryPlugin/AWSAPIPlugin.swift

```diff
@@ -38,6 +37,23 @@ final public class AWSAPIPlugin: NSObject, APICategoryPlugin {
     /// and is clearable by `reset()`. This is implicitly unwrapped to be destroyed when resetting.
     var subscriptionConnectionFactory: SubscriptionConnectionFactory!
 
+    /// The underlying map that retains instances of NetworkReachabilityNotifier.  Using a computed property
+    /// to work around @available for use on stored properties
+    var iReachabilityMap: [String: Any]?
+
+    @available(iOS 13.0, *)
+    var reachabilityMap: [String: NetworkReachabilityNotifier] {
+        get {
+            if iReachabilityMap == nil {
+                iReachabilityMap = [String: NetworkReachabilityNotifier]()
+            }
+            return iReachabilityMap as! [String: NetworkReachabilityNotifier]
```

Comment:
Not super happy about this, but attempting to work around the whole iOS 13 `@available` compatibility